### PR TITLE
Refactoring to delegate things to `store` method.

### PR DESCRIPTION
**Pull request by americodls**
I made a refactoring to delegating and aliasing some methods. IMHO these changes avoid duplications and improve the readability.

If you agree, enjoy it. If not, please, give me some feedback to aggregate to my knowledge.

Best regards.

### CODE REVIEW

File: lib/request_store.rb [hunk 1]
```diff
@@ -1,42 +1,29 @@
...
 
 module RequestStore
-  def self.store
-    Thread.current[:request_store] ||= {}
-  end
-
-  def self.clear!
-    Thread.current[:request_store] = {}
-  end
+  class << self
```
Comment (steveklabnik) [inline lib/request_store.rb]: I really dislike this style, personally. I find it significantly less readable.

**Comment by steveklabnik** [comment]
Hey thanks! I'm not entirely sure that it's worth it, but I guess `Forwardable` is in the standard library...

**Comment by americodls** [comment]
Thanks for your comments. I agree with them, so I made some more adjustments.
It looks better now?

**Comment by steveklabnik** [comment]
I think ultimately I prefer the straight-up implementation. Thank you though!

**Comment by steveklabnik** [comment]
(If this were a more complex project, I might consider it)

**Pull request closed without merging**